lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -248,7 +256,7 @@ struct CircuitLoweringState {
 
   // Record the set of remaining annotation classes. This is used to warn only
   // once about any annotation class.
-  StringSet<> alreadyPrinted;
+  StringSet<> alreadyProcessed;
```

Comment:
Please put something about "annotation" in the name of this, and mention that these are guarded by the mutex.